docs/interfaces/postgres.rst

```diff
@@ -340,39 +417,54 @@ execution engine address different requirements (see :ref:`Clustering
 
 The listed features below cover the main differences in implementation and
 dialect between CrateDB and PostgreSQL. A detailed comparison between CrateDB's
-SQL dialect and standard SQL is defined in
-:ref:`appendix-compatibility`.
+SQL dialect and standard SQL is defined in :ref:`appendix-compatibility`.
 
-``COPY``
---------
 
-CrateDB does not support the distinct sub-protocol that is used to serve
-``COPY`` operations and provides another implementation for transferring bulk
-data using the :ref:`sql-copy-from` and :ref:`sql-copy-to` statements.
```

Comment:
Why is this section removed? It's still valid and important to note.

Reply:
This section moved to https://github.com/crate/crate/pull/11631/files#diff-914d5e25edd73f4ac2ed45cec9bc9b35c59328b83b15cdee5d934613665778b3L100
But it is really to track all this, because the pr is so large.

Reply:
Ah ok thx, but I still think it should stay here inside the `Client` section as the implementation detail difference affects client behaviour. Or we keep an entry here as well and link it.

Reply:
Fixed with 2b105000.